src/app/rundown-view/components/offsetable-part/offsetable-part.component.scss

```diff
@@ -28,15 +28,8 @@
       right: 0;
       bottom: 0;
       left: 0;
-      background: repeating-linear-gradient(
-                      120deg,
-                      #00000000,
-                      #00000000 8px,
-                      #ffff0055 8px,
-                      #ffff0055 16px
-      );
+      background: repeating-linear-gradient(120deg, #00000000, #00000000 8px, #ffff0055 8px, #ffff0055 16px);
```

Comment:
We can't use variables here?

Reply:
Maybe we should make the gradiant itself a variable instead of having each individual color as a gradiant?
https://www.shecodes.io/athena/10807-css-gradients-using-css-variables

Reply:
That way it's working, Done